base.py

```diff
@@ -10,26 +10,5 @@ def base():
     return render_template('base.html')

```

Comment:
I think this file should be deleted completely. Why is it here?

Reply:
well, thats a good question